crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1557,9 +1546,9 @@ impl AuthorityStore {
         }
 
         let tombstones = effects
```

Comment:
Line 1149 [https://github.com/MystenLabs/sui/blob/d4db615eccfcfcfaec6ebee166f2dccdc895f8b2/crates/sui-core/src/authority/authority_store.rs#L1160](url) is another place where tombstones are gathered. I wonder if there can be a unified function to be used in both places.

Reply:
Yeah that one is a bit harder because it makes a distinction between deleted and wrapped tombstone.
We may consider doing another refactoring where we no longer makes such distinction